base/compiler/abstractinterpretation.jl

```diff
@@ -59,7 +59,7 @@ function abstract_call_gf_by_type(interp::AbstractInterpreter, @nospecialize(f),
             # as we may want to concrete-evaluate this frame in cases when there are
             # no overlayed calls, try an additional effort now to check if this call
             # isn't overlayed rather than just handling it conservatively
-            matches = find_matching_methods(arginfo.argtypes, atype, method_table(interp),
+            matches = find_matching_methods(ipo_lattice(interp), arginfo.argtypes, atype, method_table(interp),
```

Comment:
```suggestion
            matches = find_matching_methods(typeinf_lattice(interp), arginfo.argtypes, atype, method_table(interp),
```
since `arginfo.argtypes` should be converted to locally valid lattice elements?

Reply:
I suppose that's right. The input lattice elements are in the typeinf lattice - only the output ones are in the IPO lattice.